src/Domain/Stager.php

```diff
@@ -61,11 +85,11 @@ public function stage(array $command, string $stagingDir): void
      */
     private function validateCommand(): void
```

Comment:
Should we say this `@throws \InvalidArgumentException`?

Reply:
I think this is the same question as before?